src/tweenable.rs

```diff
@@ -1100,6 +1209,28 @@ impl<T> Delay<T> {
     pub fn clear_completed_event(&mut self) {
         self.event_data = None;
     }
+
+    /// Enable running a one-shot system upon completion.
+    ///
+    /// If enabled, the tween will run a system via a provided [`SystemId`] when the
+    /// animation completes. This is similar to the [`with_completed()`],
+    /// but uses a system registered by [`register_system()`] instead of a callback.
+    ///
+    /// See [`with_completed_system()`] for details.
+    ///
+    /// [`set_completed_system()`]: Tween::set_completed_system
```

Comment:
You never need to tell rustdoc about the current function being documented. You do however need to help rustdoc with other functions:

```suggestion
    /// [`with_completed_system()`]: Tween::with_completed_system
```